Cilsil/Cil/Parsers/StlocParser.cs

```diff
@@ -79,12 +79,20 @@ internal class StlocParser : InstructionParser
             else
             {
                 var variable = new LocalVariable(LocalName(index), state.Method);
+                var variableType = FindTypeOfLocalVariable(state, variable);
+                if (variableType != null)
+                {
+                    type = variableType;
+                }
+                else
```

Comment:
Why are we adding this branching which affects whether we invoke RegisterLocalVariable or not? We can just directly retrieve the type from Cecil:

![image](https://user-images.githubusercontent.com/38902361/122817438-be9ac300-d28c-11eb-90ad-3717c05d79fa.png)

Reply:
> Why are we adding this branching which affects whether we invoke RegisterLocalVariable or not? We can just directly retrieve the type from Cecil:
> 
> ![image](https://user-images.githubusercontent.com/38902361/122817438-be9ac300-d28c-11eb-90ad-3717c05d79fa.png)

We have to use this branch, but not just for the purpose of retrieving variable type. We also want to avoid duplicate or overriding the registered variable.